scripts/apps/search/directives/DateFilters.ts

```diff
@@ -16,6 +17,7 @@ export function getDateRangesByKey(): Dictionary<string, IDateRange> {
         elasticSearchDateRange: {
             lte: 'now+1M/d',
         },
+        active: false,
```

Comment:
We shouldn't have UI state outside of the UI components.

Instead, you could iterate the results of `getDateRangesByKey()` and add the property later, inside the UI view. You can look up how is it implemented in filters view.